### ISSUE TITLE
"not available" for SOLD works in Preview PDF

### ISSUE THREAD

**Issue by dertel** [dertel/old-issues#323]
**[Original report](https://bitbucket.org/davidartrecord/artrecord/issues/323) by Madeleine Paré (Bitbucket: [madeleineArtRec](https://bitbucket.org/madeleineArtRec), ).**

----------------------------------------

Can you make it so that works that have sold appear as "Not available" in the PDF? The price would also be omitted for those works.

**Comment by dertel** [comment]
**Original comment by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

When I implemented this feature, I made it so that "not available" only appears when the user has selected "show prices". So if you check "show prices" the "Not available" text should appear on sold works.

I think the idea was that availability was related to prices, and so if the user didn't want people to see the prices, they probably also wouldn't want people to know if it was sold or for sale.

Let me know if you want the "Not available" to appear no matter what options the user chooses.

xx
